### PR TITLE
Refactor to ts

### PR DESCRIPTION
**Pull request by Caitlynn**
This PR refactored the repo to typescript.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,10 @@ COPY package.json package-lock.json ./
...
 
 COPY .eslintrc ./
-COPY bin ./bin
+COPY src ./src
+COPY jest.config.js ./
+COPY tsconfig.json ./
 COPY test ./test
-COPY lib ./lib
+
+COPY .git ./.git
```
Comment (joshredwards) [inline Dockerfile:15]:
```
COPY .eslintrc jest.config.js tsconfig.json ./
COPY src ./src
COPY test ./test
COPY .git ./git
```

File: src/lib/parsing.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import _ from 'lodash';
```
Comment (joshredwards) [inline src/lib/parsing.ts:1]: Can we avoid `lodash` as a whole and use `lodash/fp` ?

**Review by joshredwards — APPROVED** [review]
Couple of comments but lgtm

**Review by kprivitera — APPROVED** [review]
looks good!

**Pull request merged**